demo/README.md

```diff
@@ -5,18 +5,18 @@ application up and running.
 
 Things you may want to cover:
 
-* Install Rails at the command prompt if you haven't yet:
+* Ruby version should be ruby 2.3.1 
```

Comment:
@rashidsp  Why Ruby version should be 2.3.1? Why can't I run this with latest Ruby?